test_scripts/API/DeleteCommand/ATF_DeleteCommand_VR_DeleteCommand_WARNINGS.lua

```diff
@@ -0,0 +1,191 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one HMI-portions
```

Comment:
@vvvakulenko, please rename the summary as the name of requirement:
"SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions"